### PR TITLE
Fix the labels on the Cluster status check page, add info to the clustering intro page

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1827.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1827.surge.sh/operations-manual/5/clustering)
- [Cluster status check](https://neo4j-docs-operations-1827.surge.sh/operations-manual/5/clustering/monitoring/status-check)

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :description: This section describes how to monitor a database's availability with the help of the cluster status check procedure.
 
-[role=label--new-5.24 label--enterprise-edition]
+[role=enterprise-edition label--new-5.24]
```
Comment (renetapopova) [inline modules/ROOT/pages/clustering/monitoring/status-check.adoc:3]: I think this should be changed to the following to render:
```suggestion
:page-role: enterprise-edition aura-db-enterprise new-5.24
```

**Review by renetapopova — COMMENTED** [review]
Looks good, just a tiny comment.

File: modules/ROOT/pages/clustering/monitoring/status-check.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/monitoring/status-check.adoc:3]: I'm not sure about Aura Enterprise. Where do you take this info from?

Comment (tselmegbaasan) [reply modules/ROOT/pages/clustering/monitoring/status-check.adoc:3]: Aura users can't use this procedure, it's an Admin procedure which is only used by the Aura operator.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1827.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1827.surge.sh/operations-manual/5/clustering)
- [Cluster status check](https://neo4j-docs-operations-1827.surge.sh/operations-manual/5/clustering/monitoring/status-check)

**Review by tselmegbaasan — APPROVED** [review]

**Pull request merged**